install/operator/pkg/cmd/internal/run/run.go

```diff
@@ -17,42 +17,43 @@
-
-	"github.com/syndesisio/syndesis/install/operator/pkg/syndesis/versions"
-	v1 "k8s.io/api/core/v1"
-
-	"github.com/syndesisio/syndesis/install/operator/pkg"
-
-	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
-	"github.com/operator-framework/operator-sdk/pkg/leader"
-	"github.com/operator-framework/operator-sdk/pkg/log/zap"
-	"github.com/operator-framework/operator-sdk/pkg/metrics"
-	"github.com/operator-framework/operator-sdk/pkg/restmapper"
-	"github.com/pkg/errors"
-	"github.com/spf13/cobra"
-	"github.com/syndesisio/syndesis/install/operator/pkg/apis/syndesis/v1beta1"
-	"github.com/syndesisio/syndesis/install/operator/pkg/cmd/internal"
-	"github.com/syndesisio/syndesis/install/operator/pkg/syndesis/configuration"
-	"github.com/syndesisio/syndesis/install/operator/pkg/util"
-	"k8s.io/apimachinery/pkg/util/intstr"
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
-	"sigs.k8s.io/controller-runtime/pkg/manager"
-
-	sdkVersion "github.com/operator-framework/operator-sdk/version"
-	"github.com/syndesisio/syndesis/install/operator/pkg/apis"
-	"github.com/syndesisio/syndesis/install/operator/pkg/controller"
-	"github.com/syndesisio/syndesis/install/operator/pkg/openshift"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
-
-	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
+    "context"
+    "fmt"
+    "k8s.io/client-go/rest"
+    "runtime"
+
+    "github.com/prometheus/client_golang/prometheus"
+
+    kubemetrics "github.com/operator-framework/operator-sdk/pkg/kube-metrics"
+    customMetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+
+    "github.com/syndesisio/syndesis/install/operator/pkg/syndesis/versions"
+    v1 "k8s.io/api/core/v1"
+
+    "github.com/syndesisio/syndesis/install/operator/pkg"
+
+    "github.com/operator-framework/operator-sdk/pkg/k8sutil"
+    "github.com/operator-framework/operator-sdk/pkg/leader"
+    "github.com/operator-framework/operator-sdk/pkg/log/zap"
+    "github.com/operator-framework/operator-sdk/pkg/metrics"
+    "github.com/operator-framework/operator-sdk/pkg/restmapper"
+    "github.com/pkg/errors"
+    "github.com/spf13/cobra"
+    "github.com/syndesisio/syndesis/install/operator/pkg/apis/syndesis/v1beta1"
+    "github.com/syndesisio/syndesis/install/operator/pkg/cmd/internal"
+    "github.com/syndesisio/syndesis/install/operator/pkg/syndesis/configuration"
+    "github.com/syndesisio/syndesis/install/operator/pkg/util"
+    "k8s.io/apimachinery/pkg/util/intstr"
+    "sigs.k8s.io/controller-runtime/pkg/client/config"
+    "sigs.k8s.io/controller-runtime/pkg/manager"
+
+    sdkVersion "github.com/operator-framework/operator-sdk/version"
+    "github.com/syndesisio/syndesis/install/operator/pkg/apis"
+    "github.com/syndesisio/syndesis/install/operator/pkg/controller"
+    "github.com/syndesisio/syndesis/install/operator/pkg/openshift"
+    logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+    "sigs.k8s.io/controller-runtime/pkg/manager/signals"
 )
```

Comment:
Can the spacing me modified so that it conforms to the original file? Much easier to see the changes then and avoid future changes changing the spacing back again.